### ISSUE TITLE
Homecoming card images

### ISSUE THREAD

**Issue by jamieadkins95** [GwentCommunityDevelopers/gwent-data-release#14]
Waiting on CDPR to provide card images. Should be ready within the next few days.

**Comment by jamieadkins95** [comment]
Card images are up, but a few are missing (e.g. alzurs thunder) will ask CDPR

**Comment by teddybee-r** [comment]
unrelated but I didnt want to open a new issue.
cards.json
shupe has 6x mage tokens but only 5x hunter and 5x knight

![image](https://user-images.githubusercontent.com/15222466/47498627-170c2f00-d85e-11e8-8908-1150c412f1cd.png)

**Comment by jamieadkins95** [comment]
Thanks for reporting! I've just fixed an issue with some missing tokens, including Shupe.

**Comment by jamieadkins95** [comment]
Still having some issues with missing card images, I'm in conversation with CDPR. Hopefully won't be too much longer

**Comment by jamieadkins95** [comment]
Missing card images are fixed:

https://github.com/GwentCommunityDevelopers/gwent-data-release/releases/tag/v1.0.0.15-3
